Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -60,7 +60,7 @@ internal struct AddressViewModel {
         case "GB":
             viewModel.labels[.city] = .cityTownFieldTitle
             viewModel.placeholder[.city] = .cityTownFieldPlaceholder
-            viewModel.schema = [.houseNumberOrName, .street, .city, .postalCode, .stateOrProvince].map { .item($0) }
+            viewModel.schema = [.houseNumberOrName, .street, .city, .postalCode].map { .item($0) }
```

Comment:
why the schema change?

Reply:
UK doesn't have province.

Reply:
in that case, is it added for example for US dynamically?

Reply:
it is there by default, just hidden for UK.
For US and Brazil it is a dropDown

Reply:
👍